spec/services/export_places_with_votes_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+
+RSpec.describe ExportPlacesWithVotes, type: :service do
+  describe "#tsv_data" do
+    let!(:place) { FactoryBot.create(:place, geograph_id: 7, lat: 51.4073, lon: -0.655554, image_uri: "000340_a49458b3.jpg", active_on_geograph: true) }
+    let!(:vote_1) { FactoryBot.create(:vote, place: place, rating: 4) }
+    let!(:vote_2) { FactoryBot.create(:vote, place: place, rating: 6) }
+    let!(:vote_3) { FactoryBot.create(:vote, place: place, rating: 3) }
```

Comment:
Could we expand out this test example a bit to cover the quite complicated SQL we have here? EG: make sure we're not including places active_on_geograph != true

Reply:
Updated! @james